cypress/integration/Signup.feature

```diff
@@ -3,6 +3,8 @@ Feature: Signup
 
     I want to be able to sign up
 
+    # NB these tests are order-sensitive, as some rely on the signup state
```

Comment:
just a thought - this comment suggests there needs to be a `background` section to do this. I guess not practical yet, but does need some thought.